src/securechip/securechip.c

```diff
@@ -692,14 +692,14 @@ static ATCA_STATUS _write_data_slot_block(uint8_t* bytes, uint16_t slot, uint8_t
 bool securechip_u2f_counter_set(uint32_t counter)
 {
     data_9_0_t data = {0};
-    ATCA_STATUS result = _read_data_slot_block(&data.bytes, SECURECHIP_SLOT_DATA0, 0);
+    ATCA_STATUS result = _read_data_slot_block(&data.bytes[0], SECURECHIP_SLOT_DATA0, 0);
```

Comment:
seems unrelated, can it be in a different PR? then there is less stuff to test per PR ;)

which linter complained? ;)

Reply:
The normal compiler complained. For some reason before the refactoring `-Wno-incompatible-pointer-types` got applied to our files even though I think the purpose was to apply it to cryptoauthlib.

Reply:
> seems unrelated, can it be in a different PR? then there is less stuff to test per PR ;)

Then this PR will never pass CI. So I don't think so.

Reply:
technically it would if the other pr would be merged first :P

but i see now, so it's ok, i will take care to test it :)